state/migration_import.go

```diff
@@ -280,6 +280,11 @@ func modelConfig(attrs map[string]interface{}) (*config.Config, error) {
 		attrs[config.SecretBackendKey] = config.DefaultSecretBackend
 	}
 
+	// Remove obsolete and no longer supported syslog forward config.
+	for _, a := range []string{"logforward-enabled", "syslog-host", "syslog-ca-cert", "syslog-client-cert", "syslog-client-key"} {
```

Comment:
We might need to implement a warning somewhere. I guess people will notice they are no longer receiving logs. But if we warn to logs will they see that because we are no longer sending logs.. lol.

Reply:
This was done 8 years ago and I highly doubt anyone is using it. We can include it in the doc for 4.0